try {
    console.log("Start");
    console.log("Finsih");
} catch (error) {
    console.log("There is an error, so this code wont't be running");
}

/*Output:
Start
Finish
*/
